### ISSUE TITLE
Problem updating Series - turns out to re-draw initial line

### ISSUE THREAD

**Issue by superfeedboy** [apexcharts/apexcharts.js#570]
Hi there,

I'm having quite a big problem with updating series.
My chart has 4 series with the same timeframe as Xaxis.

Initial draw is OK but when i zoom in, i want to draw an updated version of each series.

At first, chart contains points grouped in average/min/max. When i zoom, i get updated series with more precise points. There for, it can go from 10 points ein the area to a hundred, regarding on the area zoomed.

The problem is that, when i zoom in, it seems to draw my updated series and immediatly draw 1 series of the initial chart!

Here is the drawing function i call twice: initialy to first draw the chart and then to update it when zoomed:
`function draw_chart(options, method) {
  if (method == "draw") {
    var chart = new ApexCharts(document.querySelector("#canvas_multiping"), options);
    $.when(chart.render()).then(function(data, textStatus, jqXHR) {
      $("#loader_24").fadeOut("fast");
    });
  } else if (method == "update") {
    var latency_max = options.series[0].data;
    var latency_average = options.series[1].data;
    var latency_min = options.series[2].data;
    var loss = options.series[3].data;
    //console.log(options.series[0].data); //1556851060000 -> loss
    ApexCharts.exec('multiping', 'updateOptions', {
      options
    });

    ApexCharts.exec('multiping', 'updateSeries', [{
      series: [{
        name: "Max",
        type: 'line',
        data: latency_max
      },{
        name: "Moyenne",
        type: 'area',
        data: latency_average
      },{
        name: "Min",
        type: 'line',
        data: latency_min
      },{
        name: "Perte de paquets",
        type: "column",
        data: loss
      }]
    }]);
  }
}`

Can you please help me on this issue as it's really putting a stop to my devs right now :'(

Best regards,
Gilles

**Comment by junedchhipa** [comment]
Can I get a codepen / codesandbox working example so I can quickly reproduce it?

**Comment by superfeedboy** [comment]
Hum i'm actually having problem reproducing the same error as my datas should be dynamic (stored in Mysql DB) :/

[https://codesandbox.io/s/34py4jnyn5?fontsize=14](https://codesandbox.io/s/34py4jnyn5?fontsize=14)

**Comment by superfeedboy** [comment]
Looks like the error is quite the same though! I have multiple series and when i zoom in it bounces and show only ONE line (max latency) and is not zoomed!

Hope it'll act the same for you so you can see what's my point ^^

Regards,
Gilles

**Comment by junedchhipa** [comment]
Thank you for providing a working example. I will look into it soon.

**Comment by junedchhipa** [comment]
Hi!
I see that you're calling `updateSeries` and `updateOptions` at the same time.
If you are calling `updateOptions`, you don't need to call `updateSeries` separately as `updateOptions` can also update the series.

Please try calling just 1 method i.e `updateOptions` and let me know if it worked.

**Comment by superfeedboy** [comment]
Hi there,

Thank you for your feedback!
I just changed to give it a try and it's OK!

But now i've a new problem appearing: all my charts are updated (even though i give the ID of the only one i want to update) :'(

**Comment by junedchhipa** [comment]
Can you open a new ticket for it?
Also, please create a codesandbox demo of the issue in the new ticket
